src/api/handlers/community.py

```diff
@@ -36,6 +36,7 @@ def registerRoutes(self) -> None:
     self.add("/communities.others.listForCommunityAdmin", self.list_other_communities_for_cadmin)
     self.add("/communities.listForSuperAdmin", self.super_admin_list)
     self.add("/communities.adminsOf", self.fetch_admins_of)
+    self.add("/communities.listNoPagination", self.communities_list_no_pagination)
```

Comment:
I don't think you we should need routes like this. There should be a URL parameter that sets the pagination length, which could be set to return all results in one query.

Reply:
Thank you for the feedback